scripts/scene/BuildingManager.gd

```diff
@@ -136,7 +136,7 @@ func _on_setting_door_state_input(event: InputEvent) -> void:
 		room_builder.setting_door_motion(event, camera.position, camera.zoom)
 
 func _on_confirming_room_state_entered() -> void:
-	popup.set_text(room_builder.popup_message).show()
+	popup.set_title(room_builder.popup_title).set_content(room_builder.popup_content).set_yes_text(room_builder.popup_yes_text).set_no_text(room_builder.popup_no_text).show()
```

Comment:
No need to pass the text on instantiating a popup (new_popup()). We can access these functions on through the room_builder/room_editor instead.